src/main/scala/com/atomist/util/lang/TypeScriptGenerationHelper.scala

```diff
@@ -41,7 +41,10 @@ class TypeScriptGenerationHelper(indent: String = "    ")
       case "java.util.List<com.atomist.rug.kind.java.support.PackageInfo>" => "any[]" //TODO
       case "java.util.List<com.atomist.rug.kind.service.ServiceMutableView>" => "any[]"
       case "java.util.List<com.atomist.tree.TreeNode>" => "any[]"
-      case "interface com.atomist.tree.TreeNode" => "TreeNode" // is this right?
+      case "interface com.atomist.graph.GraphNode" =>
+        // TODO for some reason, the correct return of "GraphNode" won't compile although it's imported
```

Comment:
really weird. It didn't seem worth blocking on it, but I took a look at the generated files and they looked totally fine with the correct `GraphNode` import, but just wouldn't compile.